assets/js/components/layout.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Header } from './index'
+
+const Layout = ({ isMenuOpen, handleMenu, isUserLoggedIn, children }) => (
+    <div className="min-h-screen flex flex-col justify-between relative">
+        {isUserLoggedIn ? "" : <Header isMenuOpen={isMenuOpen} handleMenu={handleMenu}/>}
```

Comment:
If'a gali apsirasyti su && 
https://dev.to/banesag/javascript-logical-operators-and-boolean-values-1l07